controller/server/controller.go

```diff
@@ -262,6 +262,10 @@ func (c *Controller) Run() {
 	if err := c.AppEnv.HostController.RegisterXWebHandlerFactory(clientApiFactory); err != nil {
 		pfxlog.Logger().Fatalf("failed to create Edge Client API factory: %v", err)
 	}
```

Comment:
PolicyEngine start was accidentally removed during API split. Adding back. My local excess API sessions were correctly removed after this was added back in

Reply:
:+1: